src/itemset.js

```diff
@@ -201,30 +227,21 @@ define([
     }
 
     function itemsetWidget(mug, options) {
-        var widget = datasources.dataSourceWidget(mug, options, "Data Source"),
+        var widget = datasources.fixtureDataSourceWidget(mug, options, "Lookup Table"),
             super_getUIElement = widget.getUIElement,
             super_getValue = widget.getValue,
-            super_setValue = widget.setValue,
-            handleChange = widget.handleChange.bind(widget),
-            labelRef = refSelect("label_ref", "Choice Label", widget.isDisabled()),
-            valueRef = refSelect("value_ref", "Choice Value", widget.isDisabled());
-
-        labelRef.onChange(handleChange);
-        valueRef.onChange(handleChange);
+            super_setValue = widget.setValue;
 
         widget.getUIElement = function () {
-            return super_getUIElement()
-                .append(labelRef.element)
-                .append(valueRef.element);
+            return super_getUIElement();
         };
 
         widget.getValue = function () {
-            var val = super_getValue();
+            var val = super_getValue(),
+                id = val.src.split(':')[val.src.split(':').length-1];
             return {
-                instance: ($.trim(val.src) ? {id: val.id, src: val.src} : null),
-                nodeset: val.query,
-                labelRef: labelRef.val(),
-                valueRef: valueRef.val()
+                instance: ($.trim(val.src) ? {id: id, src: val.src} : null),
+                nodeset: "instance('" + id + "')",
```

Comment:
isn't there often more to the nodeset? something like `instance('products')/products/product`. maybe the `/products/product` part get added somewhere I haven't seen yet...?